packages/react-instantsearch-hooks/scripts/transforms/__testfixtures__/ris-v6-to-v7/app-inline.output.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import {
+  InstantSearch,
+  Breadcrumb,
+  HierarchicalMenu,
+  ClearRefinements,
+  CurrentRefinements,
+  ToggleRefinement,
+  HitsPerPage,
+  Menu,
+  Pagination,
+  RangeInput,
+  RefinementList,
+  SearchBox,
+  SortBy,
+} from 'react-instantsearch';
+
+function App() {
+  return (
+    <InstantSearch
+      /* TODO: `searchState` is no longer supported. This is now handled via an `onStateChange` callback.
+      See https://www.algolia.com/doc/api-reference/widgets/instantsearch/react/#widget-param-onstatechange */
+      searchState={{}}
```

Comment:
these should be commented out too maybe?

Reply:
I think it's best they have a TS error that way there's even less chance they miss the `TODO`, wdyt ?